packages/@n8n/benchmark/scripts/runForN8nSetup.mjs

```diff
@@ -36,6 +36,14 @@ async function main() {
 	const vus = argv.vus;
 	const duration = argv.duration;
 
+	const hasN8nLicense = !!n8nLicenseCert || !!n8nLicenseActivationKey;
```

Comment:
Shoudl we check for tenant ID as well?

Reply:
I think that's not necessarily required, but only for certain licenses, so I left it out